src/platform_impl/windows/mod.rs

```diff
@@ -1,6 +1,11 @@
 #![cfg(target_os = "windows")]
 
-use winapi::{self, shared::windef::HMENU, shared::windef::HWND};
+use std::hash::Hash;
```

Comment:
Where is this added import used?

Reply:
Removed it, since it is in the std prelude. rust-analyzer seems to have added it? 😕